npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
@Alex-Tideman @rrgayhart @brittanystoroz We have `npm-debug.log*` in our .gitignore, but the npm-debug.log file is still pushed up to github. Do you know why that would happen/ how we can stop that? Thanks!